tests/integration/telemetry/policy/testdata/ratelimitservice.yaml

```diff
@@ -104,7 +104,7 @@ spec:
         app: ratelimit
     spec:
       containers:
-      - image: envoyproxy/ratelimit:v1.4.0
+      - image: envoyproxy/ratelimit:6f5de117 # 2021/01/08
```

Comment:
@mandarjog  @kyessenov @douglas-reid  Looks like envoyproxy/ratelimit release is not relaly maintained. envoy has dropped v2 proto version support, but the latest release from https://github.com/envoyproxy/ratelimit/releases is still 1.4.0 which can only talk v2.

In the [rate limiting doc](https://preliminary.istio.io/latest/docs/tasks/policy-enforcement/rate-limit/#global-rate-limit), we mentioned nothing about envoy rate limit service. So I think it is fine to just ping an alpha release build in the test?